esm_analysis/carbon.py

```diff
@@ -305,13 +365,23 @@ def spco2_decomposition_index(
     def regression_against_index(ds, index, psig=None):
         terms = dict()
         for term in ds.data_vars:
-            if term != 'spco2':
+            if term != decompose:
```

Comment:
What does this mean here? Shouldn't it be `if term not in ['spco2', 'fgco2']` if I understand correctly?

Reply:
I just want to skip the decompose (which can be spco2 or fgco2) variable. but if I want to decompose `fgco2` I still need to have `spco2` to get the `spco2` contribution

Reply:
Oh I see now. I didn't see where the keyword came in. Maybe you could add a comment here clarifying what would be expected here.